### ISSUE TITLE
Sending automatic email after user account creation

### ISSUE THREAD

**Issue by will-code-for-pizza** [directus/docs#40]
Hi all,
is there a solution for sending automatic mail to a new user account, including the username and password or an activation link (better security) ?

Thanks in advance.
Will Code

**Comment by rijkvanzanten** [comment]
Yeah, use the invite flow: https://docs.directus.io/app/user-directory/#invite-a-user / https://docs.directus.io/reference/system/users/#invite-a-new-user 👍🏻
